db/compaction/compaction_job.cc

```diff
@@ -1958,7 +1958,10 @@ void CompactionJob::LogCompaction() {
       stream.EndArray();
     }
     stream << "score" << compaction->score() << "input_data_size"
-           << compaction->CalculateTotalInputSize();
+           << compaction->CalculateTotalInputSize() << "oldest_snapshot_seqno"
```

Comment:
I think the oldest snapshot timestamp is pretty useful too. For example we can use it to correlate what was going on in the application logs that might've taken the snapshot.

Reply:
... But it's likely not available here on the compaction object. Anyways I'll add it in DBMonitor stats so that should be enough.